tests/e2e/helpers/CourseHelper.ts

```diff
@@ -85,3 +87,19 @@ export function deleteCourseAdmin(course: Course) {
     cy.url().should("contain", "/all-courses");
     cy.get("div").contains(course.courseName).should("not.exist");
 }
+
+export async function deleteCourses(courses: Course[], userAuth: Account) {
+    MachineUserAuthenticationManagement.setVueEnvVariable();
+    await MachineUserAuthenticationManagement._getRefreshToken(userAuth);
+
+    const course_management = new CourseManagement();
+    const existingCourses = Object.values((await course_management.getCourses()).returnValue).flat();
```

Comment:
why are you using Object.values here? it should return a Course[], which is already flat

Reply:
Good Point. I think that is a remainder from the previous implementation for accounts, where I got ```[admins[],lecturers[],students[]]``` as return value. I'll change that